src/main/resources/application.yml

```diff
@@ -56,9 +56,6 @@ aws:
   s3:
     bucket-name: yurjinia
     region: eu-north-1
-  credentials:
-    access-key-id: AKIAXYKJXJRQVQM4LYGI
```

Comment:
you need to store here keys where you locate .env file.

application.yml/credentials.access-key-id - credentials.env/access-key-id variable
application.yml/credentials.secret-access-key - credentials.env/secret-access-key variable

And then you need to inject the values of to S3Configuration fields and decrypt them there.